scripts/globals/mobskills/whispers_of_ire_self.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- Whispers of Ire
+-- Targets itself and removes all debuffs
+-- Type: Magical
+-----------------------------------
+require("scripts/globals/mobskills")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    local dispel = target:eraseStatusEffect()
+
+    if dispel ~= xi.effect.NONE then
```

Comment:
it doesn't even try if it didn't succeed in erasing at least one effect? and by doing this in the skillcheck, its already erasing one status before the mob even finishes. If someone stuns it, one effect still got erased.